app/src/main/java/com/nutomic/syncthingandroid/activities/FolderActivity.java

```diff
@@ -304,12 +324,18 @@ public void onPause() {
     @Override
     protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
-        outState.putBoolean(IS_SHOWING_DELETE_DIALOG, mDeleteDialog != null && mDeleteDialog.isShowing());
+        outState.putBoolean(IS_SHOW_DELETE_DIALOG, mDeleteDialog != null && mDeleteDialog.isShowing());
         Util.dismissDialogSafe(mDeleteDialog, this);
 
+        outState.putBoolean(IS_SHOW_OVERRIDE_REMOTES_DIALOG, mOverrideChangesDialog != null && mOverrideChangesDialog.isShowing());
```

Comment:
```suggestion
        outState.putBoolean(IS_SHOW_OVERRIDE_CHANGES_DIALOG, mOverrideChangesDialog != null && mOverrideChangesDialog.isShowing());
```

Please change in all places. If unclear, it could also be `OVERRIDE_REMOTE_CHANGES` (but not just `REMOTES`).

Reply:
> Please change in all places. If unclear, it could also be `OVERRIDE_REMOTE_CHANGES` (but not just `REMOTES`).

I missed these, reverted them as well in the latest push.